### PR TITLE
ddcutil: add page

### PR DESCRIPTION
**Pull request by franga2000**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 1.2.0

### CODE REVIEW

**Review by einverne — DISMISSED** [review]
Thank you for the new page. There are two small changes that need to by applied.

File: pages/linux/ddcutil.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ddcutil
+
+> Control the settings of connected displays via DDC/CI.
+> More information: <https://www.ddcutil.com/>.
```
Comment (navarroaxel) [inline pages/linux/ddcutil.md:4]:
```suggestion
> More information: <https://www.ddcutil.com>.
```

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/ddcutil.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# ddcutil
+
+> Control the settings of connected displays via DDC/CI.
+> More information: <https://www.ddcutil.com>.
```
Comment (marchersimon) [inline pages/linux/ddcutil.md:4]: Do you think we should add this?
```suggestion
> This command requires the kernel module `i2c-dev` to be loaded. See `modprobe` for more details.
> More information: <https://www.ddcutil.com>.
```

File: pages/linux/ddcutil.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- List all compatible displays:
+
+`ddcutil detect`
+
+- Change the brightness (option 0x10) of display 1 to 50%:
+
+`ddcutil -d {{1}} setvcp {{10}} {{50}}`
```
Comment (marchersimon) [inline pages/linux/ddcutil.md:12]: We use long option when possible. Please also change this in the other examples:
```suggestion
`ddcutil --display {{1}} setvcp {{10}} {{50}}`
```

File: pages/linux/ddcutil.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Increase the contrast (option 0x12) of display 1 by 5%:
+
+`ddcutil -d {{1}} setvcp {{12}} {{+}} {{5}}`
+
+- Read the settings of display 1:
+
+`ddcutil -d {{1}} getvcp ALL`
```
Comment (marchersimon) [inline pages/linux/ddcutil.md:20]:
```suggestion
`ddcutil -d {{1}} getvcp {{ALL}}`
```

File: pages/linux/ddcutil.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/ddcutil.md:4]: Perhaps this better:
```suggestion
> This command requires the kernel module `i2c-dev` to be loaded. See also: `modprobe`.
> More information: <https://www.ddcutil.com>.
```

**Review by marchersimon — APPROVED** [review]
Thanks @franga2000

**Comment by marchersimon** [comment]
I'll dismiss ＠einverne's review, since the comments have been addressed.

**Pull request merged**